app/serverless/aws-lambda/provider.go

```diff
@@ -104,7 +104,11 @@ func (p Provider) Deploy(nsConfig api.Config, details app.Details, userConfig ma
 }
 
 func (p Provider) Exec(ctx context.Context, nsConfig api.Config, details app.Details, userConfig map[string]string) error {
-	return fmt.Errorf("exec is not supported for the aws-lambda provider")
+	return fmt.Errorf("exec is not implemented for the aws-lambda provider yet")
```

Comment:
My assumption is that the change to this comment indicates that this can be done in the future? I didn't think you could shell into a lambda (but this was entirely speculation) If you can, this is really cool, although I'm not sure how useful. :)

Reply:
My thinking is that `nullstone exec` could literally run the lambda function with an input payload.
Although, we may want to call this `nullstone run` instead.